src/java.base/share/classes/java/lang/runtime/ExactnessMethods.java

```diff
@@ -32,7 +32,7 @@
  *
  * @since 22
  */
-public class ExactnessMethods {
+final public class ExactnessMethods {
```

Comment:
I think it should be `public final` (order of modifier)

Reply:
Done